content/articles/data-entry-support-with-html/index.md

```diff
@@ -0,0 +1,81 @@
+
+* Automatic Collection of Data
+Apart from CRMs, you would possibly additionally have databases in which the statistics want to be entered. Instead of manually coming into the statistics into the databases, you'll, without difficulty, automate the procedure by directing the statistics from the bureaucracy to the databases. This reduces the statistics access mistakes to a massive volume. You may be assured of the consistency of statistics as there may be no alternate is statistics layout or spellings.
+
+### Creating and Submitting HTML form to Google Sheets.
+Below is how to create a data entry sheet with Google Apps Script.
+**Step One:** Go to https://drive.google.com/drive/my-drive and login into your Google Drive account. 
+A Google Account is a necessity to use this service, so register for one on that site if you don't have one.
+
+**Step Two:** After logging in to Google Drive, there is a New button on the top left of the page. Click on it and choose the Google sheet alternative and Blank Spreadsheet option.
+
+**Step Three:** Give your sheet an appropriate name. After giving it a name, a link is created in the Drive.
+
+### Creating the HTML Form
+Controls are specific components found in forms like entry boxes, checkboxes, radio buttons, pushbuttons, etc. The form is usually filled out by altering its controls, such as inputting text, choosing items, and sending it to an internet host for processing.
+An HTML form is generated using the `<form>` element. Here's a sample of a basic sign up form:
+```html
+<form>
+    <label>Name: <input type="text"></label>
+    <label>Secret Code: <input type="password"></label>
+    <input type="submit" value="ENTER">
+</form>
+```
+
+### Sending form to Google Sheets
+Google Forms may be utilized as an application by going to the responses button and reading all responses. Whenever you're gathering a bunch of data from some individuals, though, you should put it all in a spreadsheet so you can organize and evaluate it later.
+It takes a long time to transfer information from Google Forms to Google Sheets manually. Alternatively, you may automate data transmission by connecting Google Forms to Google Sheets. It implies that when a respondent uploads a form, the data will display in the spreadsheet 
+simultaneously. 
+
+Follow the procedure below to send the form you created to Google Spreadsheet. It can be either a brand-new or already created spreadsheet.
+1. Select the `response` option in the selected form.
+2. Besides the Google Sheets symbol, select the dots box. Pick Response Location from the drop-down menu.
+3. You may choose where your reply will go from here. You have two options: creating a blank spreadsheet or selecting an already made one.
```

Comment:
3. You may choose where your reply will go from here. You have two options: creating a blank spreadsheet or selecting an already made one.

Reply:
```suggestion
3. You can choose where your reply should be saved. You have two options: Create a blank spreadsheet or select an existing already one.
```